src/index.js

```diff
@@ -1,15 +1,24 @@
-
-
 import './index.css';
-
+import React from 'react';
+import ReactDOM from 'react-dom';
+import {BrowserRouter, Route} from "react-router-dom"
+import './index.css';
+import App from './App';
 import * as serviceWorker from './serviceWorker';
-import {rerenderEntireTree} from './render'
-import {state} from './components/redux/State'
+import {state, addPost, updateNewpostText, subscribe} from './components/redux/State'
+
+const rerenderEntireTree = (state) => {
+ReactDOM.render(
+  <BrowserRouter>
+  <React.StrictMode>
+    <App appState={state} addPost={addPost} updateNewPostText={updateNewpostText}/>
+  </React.StrictMode>,
```

Comment:
The comma "," after </React.StrictMode> is redundant.

Reply:
done